google_maps/public/html/google_maps_edit.html

```diff
@@ -1,15 +1,67 @@
-<div class="wrapper-comp-settings is-active editor-with-buttons google-maps-edit-wrapper">
-  <p>Welcome to Google Maps Xblock!</p>
+{% load i18n %}
+
+<div class="wrapper-comp-settings is-active editor-with-buttons" id="google-maps-edit-wrapper">
+  <div class="xblock-inputs editor_content_wrapper">
```

Comment:
Historically, dashes were more preferable because a lot of browsers, as I understood, didn't play well with classes and ids that contained underscores.

But, at this future age, it seems we can use either of those. @mferenca can decide which one he prefers.